langkit/templates/pkg_analysis_body_ada.mako

```diff
@@ -937,6 +937,38 @@ package body ${ada_lib_name}.Analysis is
       return Last_Child_Index (Node.Internal.Node);
    end Last_Child_Index;
 
+   -----------------
+   -- First_Child --
+   -----------------
+
+   function First_Child
+     (Node : ${root_entity.api_name}'Class) return ${root_entity.api_name} is
+   begin
+      Check_Safety_Net (Node);
+
+      if Node.Children_Count = 0 then
+         return No_${root_entity.api_name};
+      else
```

Comment:
This is not necessary, as the call to `Child` below already return none in that case (at least that's what the doc mentions). Same below!